actnow/accounts/admin.py

```diff
@@ -15,11 +15,22 @@ class ActNowUserAdmin(UserAdmin):
         "is_staff",
         "is_superuser",
         "is_active",
+        "is_deleted",
     )
 
     fieldsets = (
         (None, {"fields": ("email", "username", "password", "groups")}),
-        ("Permissions", {"fields": ("is_staff", "is_superuser", "is_active")}),
+        (
+            "Permissions",
+            {
+                "fields": (
+                    "is_staff",
+                    "is_superuser",
+                    "is_active",
+                    "is_deleted",
```

Comment:
Hadn't seen this; `is_deleted` shouldn't be a permission. It should just be something that say QuerySet uses similar to how we handled it in GMMP when a story sheets was deleted.

Or may be I should ask, why are we setting it up as a permission?

Reply:
@esirK

Reply:
The >why are we setting it up as a permission?

part, there isn't any reason to have it there. I'll update this and also ensure `is_deleted` can't be set to `true` during creation.